project.clj

```diff
@@ -50,7 +50,9 @@
                  [akiroz.re-frame/storage "0.1.2"]
                  [cljsjs/chartjs "2.6.0-0"]
                  [org.web3j/core "2.3.1"]
-                 [cljs-web3 "0.19.0-0-7"]]
+                 [cljs-web3 "0.19.0-0-7"]
+                 [jdbc-ring-session "1.0"]
+                 [ring/ring-session-timeout "0.2.0"]]
```

Comment:
I think it would be nice to move these closer to the other ring-related dependencies (roughly line 20).